### PR TITLE
Fix Issue 157 and 159 & implement 158 and 101

### PR DESCRIPTION
**Pull request by apenalver2**
- Allow cleaning the selected values of the systelab-listbox #158
- Allow customization of the messages in the systelab-grid #101
- Fix: show the message in the grid if there are no values #159
- Fix: multiple selections retrieve values in listbox #157

### CODE REVIEW

File: src/app/systelab-components/grid/README.md [hunk 1]
```diff
@@ -289,6 +289,8 @@ protected getHeaderContextMenuOptions(): Array<GridContextMenuOption<string>> {
 | preferenceName | string | | Preference prefix in order to store the columns size |
 | multipleSelection | boolean | false | Set multiple selection |
 | showChecks | boolean | false | Show a column with a checkbox for each element |
+| noRowsText | string | No Rows To Show | String/Template to display in the grid if the grid hasn't got values (can be html) |
```
Comment (alfonsserra) [inline src/app/systelab-components/grid/README.md]: String/Template to display in the grid is empty (can be html)

Comment (apenalver2) [reply src/app/systelab-components/grid/README.md]: I have changed the text

**Comment by apenalver2** [comment]
I changed the code to use the default values provided by ag-grid, then if you don't set a value in the template which can be different depending on the grid and the information, will use the default values provided by ag-grid.

**Comment by apenalver2** [comment]
Thanks

**Pull request merged**